pkg/kubelet/qos/util/qos.go

```diff
@@ -46,6 +47,24 @@ func isResourceBestEffort(container *api.Container, resource api.ResourceName) b
 	return !hasReq || req.Value() == 0
 }
 
+// GetQosPod returns the QoS class of a pod
+func GetQosPod(pod *api.Pod) string {
```

Comment:
nit: Also add some comments that describes the underlying policy (lowest class among all containers), since this method will be linked into many places.